view/head.tpl

```diff
@@ -3,13 +3,14 @@
 <link rel="stylesheet" type="text/css" href="$stylesheet" media="all" />
 
 <!--[if IE]>
+<!--
+@TODO Dead file
 <script type="text/javascript" src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+//-->
 <![endif]-->
 
-<link rel="shortcut icon" href="$baseurl/images/friendica-32.png" />
+<link rel="shortcut icon" href="images/friendica-32.png" />
 
-<script type="text/javascript" src="$baseurl/include/jquery.js" ></script>
-<script type="text/javascript" src="$baseurl/include/main.js" ></script>
+<script type="text/javascript" src="js/jquery.js?v=1.4.2" ></script>
+<script type="text/javascript" src="js/main.js" ></script>
```

Comment:
There is  `<base href="$baseurl" />` in line 2 which sets the base URL for all other URLs. I can test it at home where it will definedly be in a sub folder.